subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -56,6 +56,7 @@
 
 #define SDU_MAX_DRIFT_PPM 100
 #define SUB_INTERVAL_MIN  400
+#define FT_MAX 255
```

Comment:
As the maximum supported FT can be set as an IXIT during qualification, shouldn't this be a KConfig?